src/pages/SignIn.tsx

```diff
@@ -0,0 +1,61 @@
+import { useForm, SubmitHandler } from 'react-hook-form'
+import './../css/form.css'
+import { Link } from 'react-router-dom'
+
+type Inputs = {
+  name: string
+  email: string
+  password: string
+  emailRequired: string
+}
+
+type SignUpProps = {
+  signIn: (input: User) => void
+}
+
+function SignIn({ signIn }: SignUpProps) {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<Inputs>()
+  const onSubmit: SubmitHandler<Inputs> = (data) => signIn(data)
+
+  watch('email')
+
+  return (
+    <div className="form-container">
+      <h2>Sign in</h2>
+      <form className="form" onSubmit={handleSubmit(onSubmit)}>
+        <div>
+          <label>email:</label>
+          <input
+            defaultValue="enter your email"
```

Comment:
These make a lot more sense as a placeholder
```suggestion
            placeholder="enter your email"
```